app/helpers/elmo_form_builder.rb

```diff
@@ -15,28 +17,41 @@ def field(field_name, options = {})
     # options[:read_only] must be true if form_mode is show
     # it may optionally be true if specified by the user
     # else it is false
-    options[:read_only] ||= @template.read_only
+    if form_mode == :show
+      options[:read_only] = read_only?
+    else
+      options[:read_only] ||= read_only? if options[:read_only].nil?
```

Comment:
Shouldn't this just be `options[:read_only] = read_only? if options[:read_only].nil?`. If the form_mode is :show we still want to obey the field-level read_only option if it is given, right?